scipy/stats/_stats_py.py

```diff
@@ -9691,9 +9785,59 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     >>> wasserstein_distance([3.4, 3.9, 7.5, 7.8], [4.5, 1.4],
     ...                      [1.4, 0.9, 3.1, 7.2], [3.2, 3.5])
     4.0781331438047861
-
+    >>> wasserstein_distance([[0, 2, 3], [1, 2, 5]], [[3, 2, 3], [4, 2, 5]])
```

Comment:
We don't have to improve the old documentation, but it's not a bad idea to describe what the code is doing.
```suggestion
Compute the Wasserstein distance between two three-dimensional samples,
each with two observations.

    >>> wasserstein_distance([[0, 2, 3], [1, 2, 5]], [[3, 2, 3], [4, 2, 5]])
```